desktop/src/main/java/bisq/desktop/common/notifications/linux/LinuxNotifications.java

```diff
@@ -33,20 +33,19 @@ public static boolean isSupported() {
 
     @Override
     public void notify(String title, String message) {
-        List<String> command = new ArrayList<String>();
+        List<String> command = new ArrayList<>();
         command.add("notify-send");
 
-        // todo add icon support
-        //command.add("-i");
-        //command.add("...");
+        command.add("-i");
+        command.add(System.getProperty("java.home") + "/../Bisq_2.png");
 
         command.add("--app-name");
         command.add("Bisq");
 
         command.add(title);
         command.add(message);
         try {
-            Runtime.getRuntime().exec(command.toArray(new String[command.size()]));
+            Runtime.getRuntime().exec(command.toArray(new String[0]));
```

Comment:
`List.toArray(T[] a)` will make a new array if `a` is no big enough, so it won't drop any argument.

I changed it to avoid warnings from the IDE. Since the warnings are useful sometimes it's worth keeping them to a minimum if possible.

Reply:
Which warning in the IDE was displayed? Do you use IntelliJ? I do not see any...